eng/Versions.props

```diff
@@ -52,8 +52,8 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- These have to be in sync with latest release branch -->
-    <VSMajorVersion>17</VSMajorVersion>
-    <VSMinorVersion>14</VSMinorVersion>
+    <VSMajorVersion>18</VSMajorVersion>
```

Comment:
This should wait, do not merge

Reply:
F# core and FCS packages versions should be bumped